src/views/VenuesList/VenueTiles/VenueTile/index.tsx

```diff
@@ -9,37 +9,45 @@ import {
 } from "@mui/icons-material";
 import GradeIcon from "@mui/icons-material/Grade";
 import PeopleAltIcon from "@mui/icons-material/PeopleAlt";
-import { useVenueTile } from "./useVenueTile.tsx";
+import { useVenueSlider } from "./useVenueSlider.tsx";
 import {
-  Capacity,
+  ArrowIcon,
   FavoriteContainer,
-  LeftArrowIcon,
-  Location,
+  LocationWrapper,
   NameContainer,
   Price,
   PriceAndLocation,
-  Rating,
-  RatingAndCapacity,
-  RightArrowIcon,
+  RatingAndCapacityContainer,
+  RatingAndCapacityWrapper,
   RowContainer,
   SlidePhotosContainer,
   VenueTileContainer,
 } from "./VenueTile.styled.tsx";
+import { useNavigate } from "react-router-dom";
 
 interface VenueTileProps {
   venue: Venue;
 }
 
 export const VenueTile: FunctionComponent<VenueTileProps> = ({ venue }) => {
-  const { name, pricePerNightInEUR, location, rating, capacity } = venue;
-  const { photos, activeStep, handleBack, handleNext } = useVenueTile(venue);
+  const { name, pricePerNightInEUR, location, rating, capacity, albumId } =
+    venue;
+  const { photos, activeStep, handleBack, handleNext } =
+    useVenueSlider(albumId);
+
+  const navigate = useNavigate();
+  const handleClick = () => {
+    console.log("clicked");
+    const path = `/venues/${venue.id}`;
+    navigate(path);
+  };
```

Comment:
Avoid using redirects in the js code. Imagine that you would like to open a link in the new tab, you right-click on the link and then you click "open in new tab". With redirects in js, you would not have such possibility. So try to figure out how to use `Link` component with `ArrowIcon`